sessions/LIGHTING_TALKS.md

```diff
@@ -0,0 +1,60 @@
+## Session(s): Project Update Lightning Talks
+
+Short project updates in rapid succession. No more than 30-45 minutes at a time (6-9 updates). This can/should be done more than once at the meetings to give at least 12-18 projects exposure.
+
+**Proposed by** @flyingzumwalt
+**Led by** @flyingzumwalt
+**Duration** 30-45 minutes
+**Who Should Attend:** Everyone
+**Who has to be there in order for it to work:** People who want to give a project update.
+
+## Overview
+
+- **Objective**: Allow many projects to disseminate information across the entire set of attendees.
+- **Activity**: In a lecture format, allow a group of people to give pre-prepared 5-minute updates on their projects.
+- **Outcome**: Recordings of all the project updates and shared context for conference attendees to operate from.
+
+## Details
+
+This is a running list. Suggest more via PRs, or contact @flyingzumwalt to add something
+
+Here's a sketch of updates we could have on Day 1 and Day 2. People might want to do another session of updates during the unconference on Day 3.
+
+### Day 1, Monday
+
+| Topic/Project           | Presenter                   |
+|:-----------------------:|:---------------------------:|
+| IPFS ecosystem overview | @flyingzumwalt              |
+| js-ipfs                 | @diasdavid & @alanshaw      |
```

Comment:
@alanshaw, I'm sending out to you my lighting talk from Q1/Q2, once you review it, let's sync up on this one. Let's make it super fun! \o/